docs/drivers/drivers.md

```diff
@@ -0,0 +1,59 @@
+## Drivers Maintenance 
+
+#### How to upgrade the Nvidia drivers in your cluster
+
+
+###### Available GPU drivers version in OpenPAI
+
+```
+"384.111"  (Default Value)
+```
+
+
+###### Cluster Configuration
```

Comment:
pls add steps number to let user known he is in which steps

Reply:
Done